lib/efi_services/efi_ap.S

```diff
@@ -7,7 +7,7 @@
  */
 
 .extern cpu_num
-.extern hand_over_entry
+.extern efi_hand_over_entry
```

Comment:
what's the purpose for this change?

Reply:
There is also a global symbol hand_over_entry exists in lib/mp_init/mpinit.c.
So rename hand_over_entry in lib/efi_services to address duplicated symbol.